src/scripts/checkdags.py

```diff
@@ -0,0 +1,21 @@
+"""Python script to check for broken DAGs.
+
+This scripts is running during the initialization and startup of Airflow during deployment.
+When DAGs are broken, this results in Airflow not being able to start.
+So, broken dags are detected during deployment.
+"""
+
+from pathlib import Path
+import subprocess
+
+
+dagsfolder = Path(__file__).parent / ".." / "dags"
+
+broken_dags = []
+for dagfile in dagsfolder.glob("*.py"):
```

Comment:
You should probably do `**/*.py`.

Reply:
Good idea!

Reply:
Hmm. Gives trouble (import errors), because of the special way Airflow handles PYTHON_PATH. So I leave it for now, only importing the dags at one level deep.